clusterloader2/README.md

```diff
@@ -4,13 +4,18 @@
 
 To run ClusterLoader type:
 ```
-go run cmd/clusterloader.go --kubeconfig=kubeConfig.yaml --testconfig=config.yaml
+go run cmd/clusterloader.go --kubeconfig=kubeConfig.yaml --testconfig=config.yaml --provider=local
```

Comment:
provider=kind

right?

[same below]

Reply:
This section may not be relevant any more, I could get rid of it since we already have the kind example below? run-e2e.sh has some gce related functionality so local provider may not be applicable

Reply:
I think local here is actually fine. This section is not about kind, but in general about running clusterloader2. You need to specify a provider (it's a required flag) and `local` seems to be the most generic choice.  

On the other hand, we should clean up and document the providers. For example, I have no idea what is the difference between `local` and `skeleton` and what is the purpose of them.